person.rb

```diff
@@ -25,4 +26,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] [optional] To make the many to many association between the person and book complete using the rental we need also to add the person and date as parameters as you did ni the book class 
```suggestion
    def add_rental(person, date)
    Rental.new(person, self, date)
  end
```